app/src/main/java/com/github/polypoly/app/ui/menu/RulesStrings.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.polypoly.app.ui.menu
+
+class RulesStrings {
+    companion object {
+
+        const val RULES_BASICS_TITLE = "Basics"
+        const val RULES_BASICS_TEXT =
+            "Before starting a game, the game owner must choose the following parameters:\n\n" +
+                    "- The duration of each round (from minutes to weeks!)\n\n" +
+                    "- The number of players (from 2 up to 8) \n\n" +
+                    "- The game mode : Richest player, Last standing and Landlord!\n\n" +
+                    "- The number of rounds\n\n" +
```

Comment:
Is it logic to have a fixed number of rounds for "Last Standing" mode?

_edit: you should somehow precise that this number isn't useful for this gamemode_

Reply:
it is said in the landlord mode description